### ISSUE TITLE
Depth of Field support?

### ISSUE THREAD

**Issue by setpixel** [gkjohnson/three-gpu-pathtracer#45]
Hi Garrett!

Would it be possible to add DoF support? That would be stellar!

**Comment by gkjohnson** [comment]
Hey @setpixel! It would definitely be possible to add more realistic camera parameters for things like bokeh blur and depth of field but it isn't high on my list of priorities at the moment. Things like more complete materials models, performance adjustments, and other features will probably come first.

If this is something you're interested in for professional work I'm happy to talk about funding the addition of more complex features to the project. Or if you'd like to look into contributing it yourself you're more than welcome and I can work with you on that!

**Comment by setpixel** [comment]
Awesome! We are working on a storyboarding tool that uses threejs to create scenes to layout storyboards. Currently, we have a style that is white/gray meshes without inverse hull outlines. It does the job, but I've been wanting to change the look to something more beautiful: cleaner gray meshes with AO and maybe some really nice bounce lighting. I can show you examples. I've been following your work for a while, so when I saw this project, I got super excited!!! 

I'm definitely interested in funding the addition of more features / helping wherever I can!!!
